src/main/steps/existing-application/post.ts

```diff
@@ -43,7 +43,7 @@ export class ExistingApplicationPostController extends PostController<AnyObject>
             );
             nextUrl = HOME_URL;
           } else {
-            if (this.isAllowedToUnLinkFromCase(existingCase)) {
+            if (await this.isAllowedToUnlinkFromExistingCase(existingCase, caseworkerUserApi, req)) {
```

Comment:
the argument is the existing case, so adding it to the name seems a bit redundant

Reply:
Done